lib/jackal/utils/spec/generator.rb

```diff
@@ -0,0 +1,133 @@
+require 'bogo-cli'
+require 'fileutils'
+require 'jackal'
+
+module Jackal
+  module Utils
+    module Spec
+      class Generator < Bogo::Cli::Command
+        TEST_DIRS = 'test/specs/config'
+
+        def initialize(opts, args)
```

Comment:
change the `opts` name to something else, like `opts_orig`. no need to change later usage of `opts`.